wait/host_port.go

```diff
@@ -178,6 +193,15 @@ func (hp *HostPortStrategy) WaitUntilReady(ctx context.Context, target StrategyT
 		}
 	}
 
+	if hp.skipExternalCheck {
+		return nil
```

Comment:
What if `SkipExternalCheck()` is used without `SkipInternalCheck()`? Doesn't user expect internal check to happen in this case?